### ISSUE TITLE
FutureWarning on `import h5py`

### ISSUE THREAD

**Issue by berceanu** [ComputationalRadiationPhysics/picongpu#2896]
Should we change this
https://github.com/ComputationalRadiationPhysics/picongpu/blob/d340812641320d3a61e343a521287fd0720ba55b/lib/python/picongpu/plugins/data/requirements.txt#L3
to h5py>=2.8.0? See https://github.com/openPMD/openPMD-viewer/issues/230

**Comment by ax3l** [comment]
If it still works with 2.8.0 it's up to the user to silence it by updating.

That said, nobody should use numpy older than 1.15 due to interesting issues with the C-representation of scalar arrays ;-)

**Comment by berceanu** [comment]
Yup, that's what I mean. For example on `Hypnos` we use `numpy 1.15.2` and `h5py 2.7.1`, which results in the warning being shown. With `h5py 2.8.0` this would be solved.

```python
/opt/pkg/devel/python/3.6.2/lib/python3.6/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
```
